config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -97,29 +97,34 @@ periodics:
     preset-aws-credential: "true"
   branches:
     - master
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api"
```

Comment:
Is CAPI needed here? It wasn't in the repos removed below.

Reply:
sorry, deleted